alea/submitters/htcondor.py

```diff
@@ -715,6 +692,6 @@ def submit(self, **kwargs):
             self.wf.graph(
                 output=os.path.join(self.generated_dir, "workflow_graph.dot"), label="xform-id"
             )
-            self.wf.graph(
-                output=os.path.join(self.generated_dir, "workflow_graph.svg"), label="xform-id"
-            )
+            # self.wf.graph(
+            #     output=os.path.join(self.generated_dir, "workflow_graph.svg"), label="xform-id"
+            # )
```

Comment:
?

Reply:
Because some large submission need too long time for making the `svg` structure of workflow.